doc/nrf/releases_and_maturity/migration/migration_guide_2.6.rst

```diff
@@ -652,6 +652,87 @@ The following changes are recommended for your application to work optimally aft
 
 * For LwM2M applications, replace the :kconfig:option:`CONFIG_LWM2M_CLIENT_UTILS_DTLS_CID` Kconfig option with :kconfig:option:`CONFIG_LWM2M_DTLS_CID`.
 
+* For Mesh applications, the Sensor API has been re-written, and the previous Sensor API is deprecated.
+  A new Sensor API has been created that should be used for new applications and applications that will be continuously updated.
+  The new Sensor API is described in :ref:`bt_mesh_sensors_readme`.
+  Usage is shown in :ref:`bluetooth_mesh_sensor_client`, :ref:`bluetooth_mesh_sensor_server` and :ref:`bluetooth_mesh_light_lc` samples, as they all have been updated to use the new Sensor API.
+
+  The :kconfig:option:`CONFIG_BT_MESH_SENSOR_USE_LEGACY_SENSOR_VALUE` is set default to `=y`.
```

Comment:
Should we tell where to find documentation for the old API?

Reply:
People that has used the old API knows how it is working already, and can use the old documentation. The just want confirmation on what they have is working. The people other type of people want to know about the new API. And sometime is a good that it is hard to find API that we do not want them to use, and easy to find API we want them to use.